gittrigger/gittrigger.go

```diff
@@ -14,18 +14,18 @@ type gitTrigger struct {
 	LastCommit string
 }
 
-func newGitTrigger() (gitTrigger, error) {
+func NewGitTrigger() (gitTrigger, error) {
 	newGit := gitTrigger{Trigger: gitTriggerString}
-	err := newGit.getLatestCommit()
+	err := newGit.GetLatestCommit()
 	if err != nil {
 		return gitTrigger{}, err
 	}
 	return newGit, nil
 }
 
-func (g *gitTrigger) getLatestCommit() error {
-	cmd := exec.Command("git", "log", "-1", "--pretty=%B")
+func (g *gitTrigger) GetLatestCommit() error {
```

Comment:
Not sure if this should be exported as the result of it is already exposed via the `gitTrigger` struct as part of the object construction?

Reply:
@jbarber good spot; fixed by 77d6773da8b07082af6edf316daadde9e82e1aee